examples/sample-plugin/src/main.py

```
@@ -37,283 +37,279 @@ async def initialize(self) -> bool:
-                }
+                "success": True,
+                "data": result,
+                "metadata": {
+                    "procedure": procedure_id,
+                    "text_length": len(text),
+                    "language": self.language,
+                },
             }
-            
+
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)